### PR TITLE
Switch from outdated SVN revision numbers to SHA ids for awikstats

### PR DESCRIPTION
**Pull request by avyayv**
Fixes issue of SVN revision numbers and replaces them with SHA ids

### CODE REVIEW

File: apertium_wikistats_bot.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def getStats(rawStats, monoLang):
             wikiKey = splitFilePath[-1] + ' ' + countType
             if not monoLang:
                 countType = splitFilePath[1] + ' ' + wikiKey
-            revisionInfo = (stat['revision'], stat['last_author'])
+            revisionInfo = (stat['sha'], stat['last_author'])
             fileCounts[countType] = (stat['value'], revisionInfo, githubBlobUrl % (stat['name'], filePath))
```
Comment (sushain97) [inline apertium_wikistats_bot.py]: Can we provide the link to the direct revision now?

Comment (sushain97) [reply apertium_wikistats_bot.py]: i.e. something like https://raw.githubusercontent.com/apertium/phenny/946b76ade9b3d79af6b6f029e517973fbf63be2c/apertium_wikistats_bot.py

**Comment by avyayv** [comment]
The revision numbers are now sliced and are a link to the commit blob page.

File: apertium_wikistats_bot.py [hunk 2]
```diff
@@ -42,14 +42,15 @@ def getStats(rawStats, monoLang):
     for stat in rawStats:
         fileFormat = stat['file_kind']
         filePath = stat['path']
+        statName = stat['name']
```
Comment (sushain97) [inline apertium_wikistats_bot.py]: If you're going to declare a var, I suggest using it when it's useful, e.g. L54.

File: apertium_wikistats_bot.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 apiURL = 'http://wiki.apertium.org/w/api.php'
 statsURL = 'http://apertium.projectjj.com/stats-service/apertium-%s/?async=false'
 githubBlobUrl = 'https://raw.githubusercontent.com/apertium/%s/master/%s'
-
+githubCommitUrl = 'https://raw.githubusercontent.com/apertium/%s/%s/%s'
```
Comment (sushain97) [inline apertium_wikistats_bot.py]: The newline after these constants was nice! Can we bring that back?

**Comment by avyayv** [comment]
Fixed.

File: apertium_wikistats_bot.py (hunk 2 above)
Comment (sushain97) [reply apertium_wikistats_bot.py]: I don't see `statName` being used on L54?

File: apertium_wikistats_bot.py (hunk 1 above)
Comment (avyayv) [reply apertium_wikistats_bot.py]: whoops looks like I forgot to push my last commit :\

**Pull request merged**